### ISSUE TITLE
Helm chart index 404 not found

### ISSUE THREAD

**Issue by yaalsn** [apache/superset#31142]
### Bug description

➜  ~ helm repo add superset http://apache.github.io/superset/
```
Error: looks like "http://apache.github.io/superset/" is not a valid chart repository or cannot be reached: failed to fetch http://apache.github.io/superset/index.yaml : 404 Not Found
```

➜  ~ helm repo add superset https://apache.github.io/superset
```
Error: looks like "https://apache.github.io/superset" is not a valid chart repository or cannot be reached: failed to fetch https://apache.github.io/superset/index.yaml : 404 Not Found
```

### Screenshots/recordings

<img width="1433" alt="image" src="https://github.com/user-attachments/assets/7d143afd-91fe-41d0-bf84-c5c774ad3555">

### Superset version

master / latest-dev

### Python version

3.9

### Node version

16

### Browser

Chrome

### Checklist

- [ ] I have searched Superset docs and Slack and didn't find a solution to my problem.
- [x] I have searched the GitHub issue tracker and didn't find a similar bug report.
- [ ] I have checked Superset's logs for errors and if I found a relevant Python stacktrace, I included it here as text in the "additional context" section.

**Comment by tomasUnverdorben** [comment]
I am experiencing the same.

**Comment by JuryA** [comment]
It seems that the `gh-pages` branch is completely missing.

**Comment by nicholaslimck** [comment]
Same issue here for the last hour or so.

**Comment by richard-fairthorne** [comment]
Happy to find out this is a new issue. Same problem. 
<img width="950" alt="Screenshot 2024-11-25 at 4 12 38 AM" src="https://github.com/user-attachments/assets/8126b29a-afd4-4545-a09c-dedc0c957560">

**Comment by DhruvBundheliya** [comment]
@dosu The URL is not incorrect; there is an issue. We are no longer able to add Superset remote repo.

> The 404 Not Found error you're encountering is due to using an incorrect URL for the Superset Helm chart repository. The correct command to add the Superset Helm repository is:

**Comment by hf-13** [comment]
+1

**Comment by almightykid** [comment]
+1 same here

**Comment by SteelAlex** [comment]
+1

**Comment by tcp-dar-gor** [comment]
Same here, we're waiting for some news. :)

**Comment by srikanthdoc** [comment]
Same issue

**Comment by maximdogonov** [comment]
Same issue 

```
root@ch-stage-control-plane-ch-1-0:~# helm repo add superset https://apache.github.io/superset
Error: looks like "https://apache.github.io/superset" is not a valid chart repository or cannot be reached: failed to fetch https://apache.github.io/superset/index.yaml : 404 Not Found
root@ch-stage-control-plane-ch-1-0:~# helm repo add superset http://apache.github.io/superset/
Error: looks like "http://apache.github.io/superset/" is not a valid chart repository or cannot be reached: failed to fetch http://apache.github.io/superset/index.yaml : 404 Not Found
root@ch-stage-control-plane-ch-1-0:~# helm repo add superset http://apache.github.io/superset/
Error: looks like "http://apache.github.io/superset/" is not a valid chart repository or cannot be reached: failed to fetch http://apache.github.io/superset/index.yaml : 404 Not Found
root@ch-stage-control-plane-ch-1-0:~# helm repo add superset https://apache.github.io/superset
Error: looks like "https://apache.github.io/superset" is not a valid chart repository or cannot be reached: failed to fetch https://apache.github.io/superset/index.yaml : 404 Not Found
root@ch-stage-control-plane-ch-1-0:~# helm repo add superset http://apache.github.io/superset/ --force-update
Error: looks like "http://apache.github.io/superset/" is not a valid chart repository or cannot be reached: failed to fetch http://apache.github.io/superset/index.yaml : 404 Not Found
```

**Comment by marko-k0** [comment]
The day when we all went from superset to superupset :smile:

**Comment by richard-fairthorne** [comment]
> The day when we all went from superset to superupset 😄

I literally chose the product last night, and failed to deploy it this morning. Luckily I found this thread before deciding that it must not be maintained.

**Comment by joaovagner-meutudo** [comment]
+1

**Comment by PyroGun** [comment]
+1

**Comment by neuromantik33** [comment]
+1

**Comment by rusackas** [comment]
I restored the branch, but don't see any change from doing so yet. Besides @mistercrunch, @villebro or @craig-rueda might have ideas on what to do here.

**Comment by Valerie-CH** [comment]
@mistercrunch Might be this github deployment to republish: https://github.com/apache/superset/deployments/github-pages

**Comment by mistercrunch** [comment]
Good catch, wasn't familiar/aware we had this deployment - I just tried re-running last deploy here -> https://github.com/apache/superset/commit/1d8b19795a4fe6c903b801d78fee62ee14448c43, let's see if it fixes things

**Comment by Valerie-CH** [comment]
Looks like github pages was also turned off for the repo according to that deployment run you ran:

```
Error: Error: Failed to create deployment (status: 404) with build version 1d8b19795a4fe6c903b801d78fee62ee14448c43. Request ID 9381:3C0017:9C658F:1330DC3:6744B49F Ensure GitHub Pages has been enabled: https://github.com/apache/superset/settings/pages
```

**Comment by mistercrunch** [comment]
I think that last PR fixed it. Please confirm!

**Comment by Valerie-CH** [comment]
Looks good:

```
➜  ~ helm repo add superset http://apache.github.io/superset/
"superset" has been added to your repositories
```

Thank you!

**Comment by caligaitan** [comment]
It looks like I can only find chart version 0.13.4 in the repository. Were the previous versions removed, or is there another place to access them?

```
% helm search repo apache-superset/superset --versions
NAME                    	CHART VERSION	APP VERSION	DESCRIPTION
apache-superset/superset	0.13.4       	4.1.1      	Apache Superset is a modern, enterprise-ready b...
```

**Comment by geekmush** [comment]
We are stuck on looking for `0.12.11`

**Comment by hjkatz** [comment]
Same here, it seems the new `index.yaml` does not contain all previous published versions.

Would y'all be able to update that?

**Comment by mistercrunch** [comment]
Now restoring previous branch commit on `gh-pages` branch, will try to redeploy the latest chart after that.

**Comment by mistercrunch** [comment]
We just protected the `gh-pages` branch, and working on a change that's going to automate opening a PR against the now-protected branch here -> https://github.com/apache/superset/pull/31155

**Comment by villebro** [comment]
Thanks @mistercrunch for all the hard work fixing + improving this workflow. Yesterday we merged `0.13.5`, and the new PR + release pipeline worked flawlessly 👍

**Comment by mistercrunch** [comment]
Nice, been fiddling quite a bit in the deep end (GHA, docker, docker-compose, python dependencies bumps, ...). It's fantastic when it all works, but all this is fairly brittle, though getting better one PR at a time...

**Comment by iblancogarcia** [comment]
This issue seems to be still present: 

`$ helm repo add superset https://apache.github.io/superset
Error: looks like "https://apache.github.io/superset" is not a valid chart repository or cannot be reached: Get "https://apache.github.io/superset/index.yaml": context deadline exceeded (Client.Timeout exceeded while awaiting headers)`

https://apache.github.io/superset/index.yaml is available, but [the repo link](https://apache.github.io/superset) takes your to SuperSet home page

**Comment by hainenber** [comment]
@iblancogarcia you were getting timed out trying to fetched the Helm chart content. It could be intermittent/network issue at play here.

I can add the Helm chart with no problem.

<img width="601" alt="Image" src="https://github.com/user-attachments/assets/0189e9ce-e36c-4422-b2b5-cdb668d46291" />

**Comment by mistercrunch** [comment]
Core issue here seems fixed. Can we close it?

**Comment by hainenber** [comment]
Closed thanks to implied lazy consensus :D
